src/Document.js

```diff
@@ -187,6 +177,39 @@ export default class Document extends ParentNode {
         return null;
     }
 
+    get documentElement() {
+        return this._documentElement;
+    }
+
+    get head() {
+        let head = this._documentElement.getElementsByTagName('head').shift();
+        if (!head) {
```

Comment:
I suggest to leave this the way it is now. If we will require other way - I'll update.